File: src/main/java/features/dicomImage/data/DicomAttributes.java
package features.dicomImage.data;

import core.data.medImage.AttributeTagType;
import core.data.medImage.MirfAttributeMockup;

/**
 * Stores {@link MirfAttributeMockup} for all well-known DICOM data attributes
 */
public final class DicomAttributes{


    /**
     * todo: (avlomakin) add link to doc
     */
    public static final MirfAttributeMockup ONE_PIXEL_VOLUME = new MirfAttributeMockup("onepixelvolume", "(1234, 1234)",
            AttributeTagType.Dicom);

    private DicomAttributes() {}

}

File: src/main/kotlin/features/dicomimage/data/DicomAttributes.kt
package features.dicomimage.data

import core.data.attribute.AttributeTagType
import core.data.attribute.DataAttributeMockup

/**
 * Stores [DataAttributeMockup] for all well-known DICOM data attributes
 */
object DicomAttributes {

    /**
     * todo: (avlomakin) add link to doc
     */
    val ONE_PIXEL_VOLUME_MM3 = DataAttributeMockup<Double>("onepixelvolume", "(1234, 1234)",
            AttributeTagType.Dicom)
}

